src/dmz_devmapper.c

```diff
@@ -13,10 +13,73 @@
 #include <string.h>
 #include <errno.h>
 #include <assert.h>
+#include <kmod/libkmod.h>
 #include <asm/byteorder.h>
 
 #include <libdevmapper.h>
 
+int dmz_load_module(const char *modname, int log_level)
+{
+	struct kmod_ctx *ctx = kmod_new(NULL, NULL);
+	struct kmod_list *modlist, *itr;
```

Comment:
modlist needs to be initialized to NULL or kmod_module_new_from_lookup() gives "libkmod: kmod_module_new_from_lookup: An empty list is needed to create lookup"